### PR TITLE
A tiny addition to the per-target metrics collection

### PR DESCRIPTION
**Pull request by yfarjoun**
* collect the normalized minimum coverage in each target

### CODE REVIEW

File: src/java/picard/analysis/directed/TargetMetricsCollector.java [hunk 1]
```diff
@@ -541,7 +543,8 @@ private void calculateGcMetrics() {
                                     interval.getName() + "\t" +
                                     fmt.format(gcDouble) + "\t" +
                                     fmt.format(coverage) + "\t" +
-                                    fmt.format(coverage / this.metrics.MEAN_TARGET_COVERAGE)
+                                    fmt.format(coverage / this.metrics.MEAN_TARGET_COVERAGE)+ "\t" +
```
Comment (nh13) [inline src/java/picard/analysis/directed/TargetMetricsCollector.java]: ```)+``` -> ```) +```

File: src/java/picard/analysis/directed/TargetMetricsCollector.java [hunk 2]
```diff
@@ -541,7 +543,8 @@ private void calculateGcMetrics() {
...
+                                    fmt.format(min/this.metrics.MEAN_TARGET_COVERAGE)
```
Comment (nh13) [inline src/java/picard/analysis/directed/TargetMetricsCollector.java]: spaces

**Comment by nh13** [comment]
:1234: / :1234:

**Comment by yfarjoun** [comment]
Responded to reviewer comments, rebased and squashed. Waiting for tests to pass and will merge then.

**Pull request merged**